pkg/standalone/runfileconfig/run_file_config.go

```diff
@@ -26,18 +26,11 @@ type RunFileConfig struct {
 // Apps represents the configuration options for the apps in the run file.
 type Apps struct {
 	standalone.RunConfig `yaml:",inline"`
-	AppDirPath           string     `yaml:"app_dir_path"`
-	Env                  []EnvItems `yaml:"env"`
+	AppDirPath           string `yaml:"app_dir_path"`
 }
 
 // Common represents the configuration options for the common section in the run file.
 type Common struct {
-	Env                        []EnvItems `yaml:"env"`
+	Env                        []standalone.EnvItems `yaml:"env"`
```

Comment:
why can't `Env []EnvItems` be part of SharedRunConfig? 
That was Common will only be the embedded struct SharedRunConfig?